src/support/components/ConfirmationOverlay.tsx

```diff
@@ -28,6 +33,25 @@ const ConfirmationOverlay: FC<OwnProps> = () => {
 
   const paygSupport = (
     <>
+      <BannerPanel
+        size={ComponentSize.ExtraSmall}
+        gradient={Gradients.WarpSpeed}
+        icon={IconFont.Info_New}
+        textColor={InfluxColors.Yeti}
+      >
+        <div>
+          For critical issues, please contact the InfluxData support team at the
+          toll-free numbers{' '}
+          <a href="tel:+18559581047" rel="noopener noreferrer" target="_self">
+            +1 855-958-1047
+          </a>{' '}
+          and{' '}
+          <a href="tel:+448081966148" rel="noopener noreferrer" target="_self">
+            +44 808-196-6148
```

Comment:
Two things:

I think we should pull these phone numbers out into `shared/constants/index` so if they need to be referenced in the future, we can reference them. And if they never get referenced, I think it's still good to document what they are. It looks like we'll probably need two variables for each phone number - one for the href and one for display.

I don't think we need a target of `_self` or a rel of `noopener noreferrer` here. A `_self` target is the default behavior, and since we're not opening a blank link, we don't need the `rel`.

Reply:
totally agree with pulling the phone numbers into shared folder. 

 I added `_self` initially because I was using the `SafeBlankLink` component (which has a set `_target` property) . But like you said, since we aren't opening a new tab, there is no need for them. consider them removed